### PR TITLE
add python 3.7 support

### PR DESCRIPTION
**Pull request by danielnaumau**

### CODE REVIEW

**Comment by Donatas-L** [comment]
Mention in README which Python versions we support

**Comment by Donatas-L** [comment]
[]

**Review by mjanuszkiewicz-tt — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,7 +16,9 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
+          - "3.7"
           - "3.10"
+          - "3.11"
```
[the remark this answers is not in the document]
Comment (danielnaumau) [reply .github/workflows/ci.yml:21]: so it's enough to test 3.7 and 3.10. And 3.11 I added just in case

[the remark this answers is not in the document]
Comment (Donatas-L) [reply .github/workflows/ci.yml:21]: I still don't understand what that matrix does. Does it publish SDK under different Python versions?

[the remark this answers is not in the document]
Comment (danielnaumau) [reply .github/workflows/ci.yml:21]: nope, It just runs tests with different python versions

[the remark this answers is not in the document]
Comment (danielnaumau) [reply .github/workflows/ci.yml:21]: image

[the remark this answers is not in the document]
Comment (Donatas-L) [reply .github/workflows/ci.yml:21]: Can you add 3.8, and 3.9 (just to set our mind at ease, even if it's the same)

**Review by Donatas-L — APPROVED** [review]

**Review by MockusT — APPROVED** [review]

**Pull request merged**